maven-core/src/main/java/org/apache/maven/project/DefaultProjectBuilder.java

```diff
@@ -269,7 +273,7 @@ private DependencyResolutionResult resolveDependencies( MavenProject project, Re
         return ids;
     }
 
-    private ModelBuildingRequest getModelBuildingRequest( InternalConfig config )
+    private ModelBuildingRequest getModelBuildingRequest( InternalConfig config, File pomFile )
```

Comment:
Why is the second argument added? I can't see it being used in the `getModelBuildingRequest` method.

Reply:
Is now solved with the TransformerContext, can be removed.